src/metabase/lib/metadata.cljc

```diff
@@ -63,29 +63,33 @@
    table-id              :- ::lib.schema.id/table]
   (lib.metadata.protocols/table (->metadata-provider metadata-providerable) table-id))
 
-(defn- fields* [metadata-providerable table-id {:keys [only-active?]}]
-  (-> (lib.metadata.protocols/fields (->metadata-provider metadata-providerable) table-id)
-      (cond->> only-active?
-        (remove (fn [col]
-                  (or (false? (:active col))
-                      (#{:sensitive :retired} (:visibility-type col))))))
-      (->> (sort-by (juxt #(:position % 0) #(u/lower-case-en (:name % "")))))))
+(defn- fields* [metadata-providerable table-id {:keys [include-sensitive?]}]
+  (->> (lib.metadata.protocols/fields (->metadata-provider metadata-providerable)
+                                      table-id
+                                      {:include-sensitive? (boolean include-sensitive?)})
+       (sort-by (juxt #(:position % 0) #(u/lower-case-en (:name % ""))))))
```

Comment:
Why is it `:include-sensitive-fields?` in some places and `:include-sensitive?` in others?